build/make/package-debian.mk

```diff
@@ -8,10 +10,10 @@ CONFFILES_FILE_TMP="$(DEBIAN_CONTENT_DIR)/conffiles_"
 DEBSRC:=$(shell find "${WORKDIR}/deb" -type f)
 
 .PHONY: package
-package: debian-with-binary
+package: debian-with-binary ## Build binary and package into .deb file
 
 .PHONY: debian
-debian: $(DEBIAN_PACKAGE)
+debian: $(DEBIAN_PACKAGE) ## Package files into .deb file
```

Comment:
This and the preceding target may create some confusion. Maybe the description should mention that here no binary shall be built. I think we need to investigate why target exists. It feels like an artefact from older times...